app/src/info/WorldDataInfoGenerator.ts

```diff
@@ -342,7 +371,15 @@ export default class WorldDataInfoGenerator implements IProjectInfoItemGenerator
       }
 
       items.push(
-        new ProjectInfoItem(InfoItemType.info, this.id, 101, "Chunks", projectItem, mcworld.chunkCount, mcworld.name)
+        new ProjectInfoItem(
+          InfoItemType.info,
+          this.id,
+          WorldDataInfoGeneratorTest.chunks,
```

Comment:
`WorldDataInfoGeneratorTest.chunks` == 1 whereas before we passed 101. Intentional?